test/test_util.py

```diff
@@ -889,3 +890,26 @@ def test_ssl_wrap_socket_sni_none_no_warn(self):
         context, warn = self._wrap_socket_and_mock_warn(sock, None)
         context.wrap_socket.assert_called_once_with(sock, server_hostname=None)
         warn.assert_not_called()
+
+
+idna_blocker = ImportBlocker("idna")
+module_stash = ModuleStash("urllib3")
+
+
+class TestWithoutIdna:
```

Comment:
Because we only have on test case that uses this blocker we can put `setup_class` and `teardown_class` on `TestUtilWithoutIdna`. If there's another test case that needs it we can generalize then :)